api/Event.json

```diff
@@ -0,0 +1,1356 @@
+              "notes": "See <a href='https://bugzil.la/691151'>bug 691151</a>."
+            },
+            "firefox_android": {
+              "version_added": true,
+              "version_removed": "59",
+              "notes": "See <a href='https://bugzil.la/691151'>bug 691151</a>."
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "initEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Event/initEvent",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "17"
+              },
+              {
+                "version_added": true,
+                "version_removed": "17",
+                "partial_implementation": true,
+                "notes": "Before Firefox 17, a call to this method after the dispatching of the event raised an exception instead of doing nothing."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "17"
+              },
+              {
+                "version_added": true,
+                "version_removed": "17",
+                "partial_implementation": true,
+                "notes": "Before Firefox 17, a call to this method after the dispatching of the event raised an exception instead of doing nothing."
+              }
+            ],
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "preventBubble": {
```

Comment:
I'm not finding any page relating to https://developer.mozilla.org/docs/Web/API/Event/preventBubble

Reply:
It’s an [obsolete Mozilla-only method](https://developer.mozilla.org/docs/Web/API/Event#Obsolete_methods).